adding-a-new-scraper.md

```diff
@@ -20,9 +20,9 @@ Going forward in this guide, `TMetricDefinition` will refer to your newly create
 For every scraper type we provide validation for the configuration so that Promitor fails to start up.
 
 This requires the following steps:
-1. Create a new validator that implements `IMetricValidator<TMetricDefinition>`
-2. Hook your new validator in our validation pipeline in `MetricsValidator`
-3. Provide a unit test for every validation rule that was added
+1. Create a new validator that implements `MetricValidator<TMetricDefinition>` where `TMetricDefinition` is the definition you made in step #2 above. This validator should reside in `.\src\Promitor.Scraper.Host\Validation\MetricDefinitions\ResourceTypes`. You can look at the contents of `GenericAzureMetricValidator` for an idea of the validation inputs, steps, and outputs typical of validator implementation.
```

Comment:
Let use `ServiceBusQueueMetricValidator` as an example here instead of `GenericAzureMetricValidator` as this is a "special one".